test/mcbedrock.js

```diff
@@ -0,0 +1,32 @@
+const Registry = require('prismarine-registry')
+const collectPackets = require('./util/collectBedrockPackets')
+const assert = require('assert')
+
+async function main (version = '1.19.63') {
+  const registry = Registry(`bedrock_${version}`)
+  let loggedIn = false
+  const handlers = {
+    play_status (version, params) {
+      if (params.status === 'login_success') loggedIn = true
+    },
```

Comment:
This seems wrong, the purpose of the `loggedIn` field in the pc test is not to check that the client logged in or any other client status, but that the expected test condition is ran. So this handling can be removed and `loggedIn` should be moved into `start_game`. 

Anyway, the reason it's like this is collectPackets was supposed to throw if the all the packets in the passed `names` param weren't received, but that functionality just didn't get implemented in #1.